docs/source/release/v3.12.0/sans.rst

```diff
@@ -14,5 +14,40 @@ ILL SANS
 - New instrument definition file for the D22 instrument
 - New instrument definition file for the D11 instrument
 
+ISIS SANS
+---------
+
+.. image::  ../../images/ISIS_SANS_312.png
+   :align: right
+   :class: screenshot
+   :width: 800px
+
+New features
+############
+- BeamCentreFinder has been implemented to work with the new backend in the python script window.
+- Added find beam centre tab to SANS GUI V2.
+- Added Sum runs tab to SANS GUI V2.
+- Added Diagnostic tab to SANS GUI V2.
+
+Improvements
+############
+- Added functionality to specify q values between which to merge.
+- Grid lines are now displayed in ISIS SANS V2
+- Added the option to hide the period selection columns in the SANS GUI V2
+- Have added the functionality to show diagnostic transmission workspaces to new GUI.
+- Have added functionality to continually plot latest results to new GUI.
```

Comment:
Unless I'm mistaken, `ISIS SANS V2`, `SANS GUI V2` and `new GUI` are all referencing the same thing. Can we just use one term to avoid confusion. Unless of course I am confused and they are different!